### PR TITLE
Adds Build and Publish Steps

### PR DESCRIPTION
**Pull request by JPRuskin**

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -35,34 +35,7 @@ process {
...
-                Get-Content $_.FullName | Add-Content "$root\Output\TreasureChest\TreasureChest.psm1" -Force
-            }
-
-            Copy-Item $root\TreasureChest.psd1 $root\Output\TreasureChest -Force
-
-            if ($Version) {
-                Update-ModuleManifest -Path $root\Output\TreasureChest\TreasureChest.psd1 -ModuleVersion $Version
-            }
-
+            Build-Module -SemVer $SemVer
```
Comment (steviecoaster) [inline build.ps1:36]: Me thinks we need a check for `Build-Module` here and install the required module if not found. Unless I'm holding how to use this build script wrong now with these changes?

**Review by steviecoaster — CHANGES_REQUESTED** [review]
One nit, the rest LGTM!

File: build.ps1 (hunk 1 above)
Comment (JPRuskin) [reply build.ps1:36]: I've got a Requirements.ps1 to install prereqs - should I just run that within the build script?

File: build.ps1 [hunk 2]
```diff
@@ -18,88 +18,74 @@ param(
...
-            if (Test-Path "$root\src\nuget\tools\TreasureChest.zip") {
-                choco pack $Nuspec.Fullname --output-directory $Nuspec.directory
-            }
+            $Nuspec = Get-ChildItem $PackageSource -recurse -filter *.nuspec
+
+            Copy-Item -Path $root\LICENSE -Destination $PackageSource
 
-            else {
+            if (Test-Path "$PackageSource\tools\TreasureChest.zip") {
+                choco pack $Nuspec.FullName --output-directory $root
```
Comment (JPRuskin) [inline build.ps1:81]: I also forgot to ensure the Build action downloads this zip - will make sure that happens.

File: treasuredocs/mkdocs.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-site_name: TreasureChest Documentation
+site_name: NexuShell Documentation
 theme: readthedocs
 repo_url: https://github.com/steviecoaster/TreausureChest
```
Comment (JPRuskin) [inline treasuredocs/mkdocs.yml:3]:
```suggestion
repo_url: https://github.com/steviecoaster/NexuShell
```
(Assuming you'll be updating the repo name - having said that, I guess this `repo_url` doesn't need to be correct)

Comment (JPRuskin) [reply treasuredocs/mkdocs.yml:3]: Foldername this is in may also want changing.

**Pull request closed without merging**